src/mount.aznfs.c

```diff
@@ -13,6 +13,8 @@
 
 int main(int argc, char *argv[])
 {
+    unsetenv("BASH_ENV");
```

Comment:
One can still use the `LD_PRELOAD` environment variable.
Is using `sh` instead of `bash`, or using `execve` not an option?